cel/cos_tlv.go

```diff
@@ -0,0 +1,34 @@
+package cel
+
+import "crypto"
+
+const (
+	// CosTypeValue indicate the CELR event is a COS content
+	// TODO: the value needs to be reserved in the CEL spec
+	CosTypeValue uint8 = 88
```

Comment:
Discussed offline, we will pick 80 for now

Reply:
done